homeassistant/components/axion_dmx/__init__.py

```diff
@@ -0,0 +1,38 @@
+"""The Axion Lighting integration."""
+
+from __future__ import annotations
+
+from libaxion_dmx import AxionDmxApi
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import CONF_HOST, CONF_PASSWORD
+
+PLATFORMS: list[Platform] = [Platform.LIGHT]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
Please extend the ConfigEntry typing, see: https://developers.home-assistant.io/blog/2024/04/30/store-runtime-data-inside-config-entry

Reply:
Done it now, thanks for the provided info!